wizeline/wize-q-remix
# feat: improve UI of switch to enable/disable a question

#### What does this PR do?
This is a UI improvement of the enable/disable question feature 

#### Where should the reviewer start?
At the list question page or question detail page

#### How should this be manually tested?
As an admin, at the list question page, you should see a switch at each Question Card Row
As an admin if the question is enabled, after clicking on the switch the question should be disabled
As an admin if the question is disabled, after clicking on the switch the question should be enabled

#### Screenshots
<img width="683" alt="image" src="https://user-images.githubusercontent.com/97201781/210649819-6750aa79-6f5d-4d90-9fa8-fe0558e574a4.png">
